### DISCUSSION TITLE
Organisation structure

### DISCUSSION THREAD

**Discussion by M-J-Robbins** [email-markup-consortium/email-markup-consortium#46] [General]
As this group grows, we need to have some more structure in place and some more formalised processes.

In [the membership forms](https://docs.google.com/forms/d/e/1FAIpQLSdr0PEwMahURCS6BSg3tMgw5yN6Yz24yJvbHypcn_OK0y94Qw/viewform) that @alicemeichi did we have 3 levels of membership

* **Supporter**
* **Member**
* **Administrator**

Anyone can be a supporter and no approval is needed so that one is easy.  But the member and administrator roles need more clarification.

### What is the approval process?
If someone wants to join as a member what is the process?
We need a set of basic criteria.  What is that criteria beyond a good knowledge of email markup? 
If someone matches the criteria do they get accepted or do we need to approve beyond that.

### Do we need to cap numbers?
There are pros and cons of having a lot of members. A lot of people could get a lot done, but also a lot more voices can slow down decision-making and add extra complexity to organising meetings.

### What are the roles and permissions
Anyone at any level can open a PR, make a suggestion or add a comment. But only members can attend the fortnightly meetings.

But I think any additional permissions should be left to admin roles.

### What is the approval process?
I'd say admins need to be members but then have additional responsibilities.

Could be elected or could be appointed. 
Certainly, if we decide to have a chairperson they would need to be elected. 

### Do we need to cap numbers?
Yes but this might be defined on a role by role basis

### What are the roles and permissions
In my mind, these would be split into specific roles with different permissions and responsibilities.

For example 
* Website team, would maintain and update the website with permission to merge PR's.
* Standards team would, would maintain and update the compliant standards with permission to merge PR's.
* Membership team would add and if necessary remove members.

**Comment by husseinalhammad** [comment]
For Administrators, I think there is a need for a **chairperson** similar to the [chair role in Open UI](https://open-ui.org/roles#chair):

> The chairs' primary responsibilities are to facilitate weekly meetings, ensure progress towards goals is made, mentor and onboard contributors, champions, and editors, create and foster a respectful and productive atmosphere for the community, enforce the working mode.

Responsibility would include:

- Planning and running meetings
  - setting agenda
  - keep the meeting on topic
  - ensures action items are clear
  - etc.
- Keep track of progress on the separate projects and ensure projects are still working towards EMC's goals
- Enforce our code of conduct


Having someone take care of these things would also free other members to focus on getting other tasks done. So not everyone is trying to manage.

**Comment by husseinalhammad** [comment]
> What is the approval process?

This is something we need to address. We already received some applications (only from the private Slack channel so far). What is the next step to formally approve these applications?

**Comment by husseinalhammad** [comment]
> Do we need to cap numbers?

- Do other similar groups have cap numbers?
- How do we define the cap number?
- What do we do when qualified people apply even though we reached the cap number? 

I get the sense many would be happy to work on one or two of the projects, but do not really want to get involved in regular meetings and organisational matters. So the total number of members most likely won't reflect how many actually attend meetings.

**Comment by M-J-Robbins** [comment]
Also we could look at regional roles to help spread our reach and maybe add meetings to help get people involved from a wider variety of time zones. Someone could represent a region and run meetings in that time zone then they can bring those ideas forward to the admin group.

**Comment by husseinalhammad** [comment]
I think this has been largely addressed during the [last meeting](https://github.com/email-markup-consortium/email-markup-consortium/blob/main/meeting-notes/2022-02-22.md#20-minutes-organisation-structure). If anything still needs considering, please feel free to flag and discuss.
